README.md

```diff
@@ -8,6 +8,27 @@ Module to:
 * automatically generate OpenAPI (Swagger) specification for existing ExpressJS 4.x REST API applications;
 * provide Swagger UI basing on generated specification.
 
+## Additions to your package.json
```

Comment:
I would state that it is optional to change package.json so would suggest something like:

```suggestion
## (OPTIONAL) Additions to your package.json

If your service is running not at the root of the server add full base path URL to package.json
```

Reply:
fixed